app/models/theme.rb

```diff
@@ -0,0 +1,7 @@
+class Theme
+  EDUCATION_THEME_BASE_PATH = '/education-training-and-skills'.freeze
+
+  def self.taxon_path_prefixes
+    [EDUCATION_THEME_BASE_PATH]
```

Comment:
Can we add `/alpha-taxonomy` here too? Then the existing taxons won't be invalid.